### ISSUE TITLE
Seaplane operations reference data

### ISSUE THREAD

**Issue by tigert** [Juanvvc/c172p-detailed#269]
# Background

This is from OH-CTL "CAPRE Aqua Model 2200" floatplane conversion STC , which doesnt have a free license, so I won't put it out for download. Instead I will be quoting the relevant performance and operational points for you below.

## Limitations

  * Engine and Limits: 
    * Lycoming 0-360-A4A 180hp
    * Maximum normal operating power 2550 rpm (150hp)
    * Takeoff (five minutes) 2700 rpm (180hp)

  * Propeller and Limits
    * McCauley 1A200/DFA
    * Static rpm at maximum permissible throttle (ground testing I think):
      * Not over 2200 rpm, not under 2100 rpm
      * Diameter between 78 and 82 inches (overhaul sanding will eat metal away)

  * Powerplant instrumentation
    * Red Line at 2700 rpm
    * Green arc 700 rpm to 2550 rpm

  * Max gross weight floatplane: 2350 lbs
  * Center of gravity floatplane: 
    * 29.8 to 45.5 at 2350 lbs.
    * 36.5 to 45.5 at 1825 lbs or less
    * straight line variation between above points

  * No acrobatic maneuvers, including spins, approved.

  * Altitude loss in a stall recovery: 300ft

### Caution

  * Retract water rudders for fast taxiing, takeoff and landing
  * Floatplane max flaps 40 degrees (hm, isn't this max of 172 anyway? most recent 172s with higher MTOW have this limited to 30 degrees anyway)

(more to follow in a while)

**Comment by tigert** [comment]
There should be a handle to retract/extend the water rudders, placed in the cockpit floor - need to find a photo of this for reference. It has a placard that reads "Water Rudder" and "Extended" in the forward edge and "Retracted" in the aft, and a double-headed arrow in between.

**Comment by tigert** [comment]
## Emergency Procedures
  * No changes in float plane version

## Normal Procedures (suppolement to Section 4 of Pilots Operating Handbook)

### Before entering floatplane
  1. Inspect floats, struts, cables, pulleys, and fittings for dents, scratches, fraying cables, and attachments to the aircraft,
  2. Inspect each float compartment by removing the rubber caps on each standpipe and pump out any accumulated water. Reinstall caps. 
    * I don't know if we need to simulate that, 

### Before starting engine
  1. Water rudder operation - CHECK visually
  2. Water rudders - DOWN for taxiing
  3. Make sure you are CLEAR before starting engine

### Takeoff
  1. Water rudders - UP
  2. Wing flaps - set 20 DEGREES (this was news to me, on wheels its no flaps or 10° on soft runways)
  3. Control wheel - hold FULL AFT
  4. Power - advance SLOWLY to full throttle / max rpm
  5. Control wheel - MOVE FORWARD gently when bow wave moves aft of wing strut, to attain planing attitude "on the step".
    * I think this is when the sim starts to jump the nose up and down! Idea for improvement here: Looking left could move the eyepoint enough to the left side so that you can actually see the float. This would be very useful with a head tracker (try linuxtrack/opentrack with a webcam!)

  6. Control wheel - APPLY LIGHT BACK PRESSURE to lift off.
    * NOTE: To reduce the takeoff water run, the technique of raising one float out of the water can be used.
  7. Wing flaps - UP once clear of obstacles

### Before Landing
  1. Water rudders - UP
  2. Wing flaps as desired or needed

### Landing
  1. Touchdown - SLIGHTLY TAIL LOW
  2. Control wheel - HOLD FULL AFT as floatplane decelerates to taxi speed

### Balked landing (go-around)
  1. RETRACT FLAPS to 10 DEGREES immediately after applying full power. Retract fully once positive rate of climb is established.

### After landing
  1. Water rudders - DOWN

**Comment by tigert** [comment]
## PERFORMANCE (Supplement to section 5 of POH)

### Stall speed
Change is minimal (+/- 1 knot) compared to landplane version.

### Takeoff distance
Takeoff distance of floatplane is **approximately 70% more** than that of a landplane.

### Rate of climb
Sea level rate of climb is in excess of 630 feet per minute

### Cruise performance
Cruise speed of floatplane will be approximately 20 knots slower than the landplane. Due to the slower speed, range will be reduced approximately 20 percent. Due to the increased fuel consumption of the 180hp engine, endurance will also be reduced.

### Landing distance (short field)
Landing distance of a floatplane will increase approximately 10% because of no braking is available on water.

**Comment by tigert** [comment]
## AIRPLANE SYSTEMS
  * An additional fuselage "V" brace is installed between the top of the front door posts and the cowl deck.
    * These are visible on OH-CTL here: www.airliners.net/photo/MIK---Malmin/Cessna-172M/0357624/L/ and here: http://www.airliners.net/photo/MIK---Malmin/Cessna-172M/0272736/L/

**Comment by dany93** [comment]
Thanks @tigert , for these accurate data.

Measure:
(eng. 160 hp, prop. 75 inches, no flaps, < 2000 ft AGL, 85 - 90 KIAS default weather conditions)
With floats (currently, extra weight 2 x 66 lbs, drag x 1.20)
**Rate of climb 500 +/- 25 ft/mn.**
Seems plausible with 160 hp, and this propeller which seems slightly "long" (high pitch angle).

**Comment by tigert** [comment]
yeah, the fdm has a very strong "behind power curve" effect at low airspeed which is especially nasty when at maximun weight and probably even more so with the extra drag of floats. Those evil treetops are always on the way.. :P

**Comment by onox** [comment]
@tigert You can create a new issue (with 3.8 release milestone) for this "behind power curve" effect problem if you want.

**Comment by onox** [comment]
@tigert And another issue for https://github.com/Juanvvc/c172p-detailed/issues/269#issuecomment-110263195
